cost/google/cloud_sql_idle_instance_recommendations/google_sql_idle_instance_recommendations.pt

```diff
@@ -6,7 +6,7 @@ category "Cost"
 severity "low"
 default_frequency "daily"
 info(
-  version: "2.0",
+  version: "2.1",
```

Comment:
Comparing to https://github.com/flexera-public/policy_templates/pull/1048

This version matches.